src/node/in_memory_ext.rs

```diff
@@ -787,18 +793,15 @@ mod tests {
             .map(|inner| inner.current_timestamp)
             .expect("failed reading timestamp");
         assert_eq!(timestamp_before, new_timestamp, "timestamps must be same");
-        let expected_response = new_timestamp;
 
-        let actual_response = node
-            .set_next_block_timestamp(new_timestamp)
-            .expect("failed setting timestamp");
+        let response = node.set_next_block_timestamp(new_timestamp.into());
```

Comment:
should `new_timestamp` to  be equal to `timestamp_before `

Reply:
yes, it's asserted above (and also why the call fails)